apps/web/pages/api/book/event.ts

```diff
@@ -712,7 +735,12 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     await sendAttendeeRequestEmail(evt, attendeesList[0]);
   }
 
-  if (typeof eventType.price === "number" && eventType.price > 0 && !originalRescheduledBooking?.paid) {
+  if (
+    typeof eventType.price === "number" &&
```

Comment:
```suggestion
!Number.isNaN(eventType.price)
```
Method documentation
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN